README.md

```diff
@@ -139,19 +141,131 @@ fuel-core run \
 --service-name fuel-ignition-node \
 --keypair {KEYGEN_SECRET_KEY} \
 --relayer {ETHEREUM_RPC_ENDPOINT} \
---ip=0.0.0.0 --port 4000 --peering-port 30333 \
+--ip=0.0.0.0 \
+--port 4000 \
+--peering-port 30333 \
+--db-path ~/.fuel-ignition \
+--snapshot {PATH_TO_CHAIN_CONFIGURATION_FOLDER}/ignition \
+--utxo-validation \
+--poa-instant false \
+--enable-p2p \
+--bootstrap-nodes /dnsaddr/mainnet.fuel.network \
+--sync-header-batch-size 100 \
+--relayer-v2-listening-contracts 0xAEB0c00D0125A8a788956ade4f4F12Ead9f65DDf \
+--relayer-da-deploy-height 20620434 \
+--relayer-log-page-size 2000 \
+--sync-block-stream-buffer-size 30 \
+--min-gas-price 1000 \
+--starting-gas-price 1000 \
+--gas-price-change-percent 10 \
+--gas-price-threshold-percent 50
+```
+All the values used for the L1 relayer are based on Ethereum Mainnet make sure `{ETHEREUM_RPC_ENDPOINT}` is a RPC communicating to Ethereum Mainnet network.
+
+Instead of directly placing your personal values on the command we advise you to use, for example, environment variables.
+
+### From source
```

Comment:
What is the difference with "Running a Local network from source"?

Reply:
The difference is the parameters, here we sue the chain config of ignition we use the realyer address on mainnet, we don't produce blocks, we use the bootstrap peers...